.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  globals: {
+    __PATH_PREFIX__: true,
+  },
+  extends: [
+    'react-app',
+    'plugin:cypress/recommended',
+    'plugin:chai-friendly/recommended',
+    'plugin:prettier/recommended',
+  ],
+  rules: {
+    'cypress/no-unnecessary-waiting': [0],
```

Comment:
@mbifulco I've had plenty of times where defining a `wait()` of some kind was useful to simulate some user behavior- but that is a rare case. I'd say leave that rule on as a `warning` for now and we can change it if it becomes an issue.
```suggestion
    'cypress/no-unnecessary-waiting': 'warn',
```

Reply:
Happy to defer to your wisdom here.  I'm good with that!